src/Symfony/Component/HttpFoundation/Exception/InvalidHostException.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpFoundation\Exception;
+
+/**
+ * The HTTP request contains invalid host data.
+ *
+ * @author SpacePossum
+ */
+final class InvalidHostException extends AbstractHostException
```

Comment:
Indeed sorry I didn't commented on the right class... I was thinking about `AbstractHostException`.